### PR TITLE
Toelli/specific

### PR DESCRIPTION
**Pull request by tomjaguarpaw**

### CODE REVIEW

File: ADBench/plot_graphs.py [hunk 1]
```diff
@@ -89,15 +89,15 @@ def read_vals(objective, graph_files, tool):
...
+    info = [(name_to_n(utils.get_test(utils.get_fn(path.split("/")))),
+             utils.read_times(os.path.join(in_dir, path)))
+            for path in tool_files]
 
     # Sort values
-    times_sorted = sorted(time_pairs, key=lambda pair: pair[0])
-    n_vals = list(map(lambda pair: pair[0], times_sorted))
-    t_objective_vals = list(map(lambda pair: pair[1][0], times_sorted))
-    t_jacobian_vals = list(map(lambda pair: pair[1][1], times_sorted))
+    times_sorted = sorted(info, key=lambda t: t[0])
```
Comment (mikhailnikolaev) [inline ADBench/plot_graphs.py]: What about name "info_sorted"?

Comment (tomjaguarpaw) [reply ADBench/plot_graphs.py]: That's fine by me.

Comment (mikhailnikolaev) [reply ADBench/plot_graphs.py]: Ok, I will change this then

Comment (mikhailnikolaev) [reply ADBench/plot_graphs.py]: Nevermind)

**Pull request merged**